src/entity/taskslog.js

```diff
@@ -1,32 +1,30 @@
 module.exports = (nga, admin) => {
-  const task = nga.entity('taskslog');
+  const taskLog = nga.entity('taskslog');
 
   const fields = [
-    nga.field('tasksName'),
-    nga.field('tenantName'),
-    nga.field('dateAssigned'),
+    nga.field('taskName'),
+    nga.field('tenantName')
+      .isDetailLink(true),
```

Comment:
Creates a link to the edit view (although I've updated the field which links in the second pull request)